server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/GitMirror.java

```diff
@@ -240,12 +242,13 @@ protected void mirrorRemoteToLocal(File workDir, CommandExecutor executor,
                 // Add mirror_state.json.
                 changes.put(mirrorStatePath, Change.ofJsonUpsert(
                         mirrorStatePath, "{ \"sourceRevision\": \"" + id.name() + "\" }"));
-
                 // Construct the log message and log.
                 summary = "Mirror " + abbrId + ", " + remoteRepoUri() + '#' + remoteBranch() +
                           " to the repository '" + localRepo().name() + '\'';
+                final RevCommit headCommit = revWalk.parseCommit(id);
+                detail = generateCommitDetail(abbrId, headCommit);
                 logger.info(summary);
-
+                logger.info(detail);
```

Comment:
How about not logging the detail?
I'm worried about the case where the detail is so long that it might give a burden to the system.

Reply:
Agree, removed.